constants.js

```diff
@@ -33,12 +35,13 @@ const checksWhitelist = {
   'oppia': {
     [openEvent]: [claCheck, changelogCheck, branchCheck, wipCheck, jobCheck],
     [reopenEvent]: [changelogCheck, branchCheck, wipCheck, jobCheck],
-    [labelEvent]: [assigneeCheck],
+    [PRLabelEvent]: [assigneeCheck],
     [synchronizeEvent]: [mergeConflictCheck, jobCheck],
     [closeEvent]: [allMergeConflictCheck],
     [editEvent]: [wipCheck],
     [issuesLabelEvent]: [issuesLabelCheck],
-    [issuesAssignedEvent]: [issuesAssignedCheck]
+    [issuesAssignedEvent]: [issuesAssignedCheck],
+    [pushEvent]: [forcePushCheck]
   },
   'oppiabot': {
```

Comment:
Let us add all the events here as well with empty lists.

Reply:
Done.